.buildkite/pipeline.yml

```diff
@@ -184,4 +184,6 @@ steps:
             - GPG_PRIVATE_KEY
             - GPG_FINGERPRINT
             - PASSPHRASE
-          command: "goreleaser release --clean"
+          command:
```

Comment:
The change seems fine to me in that the main entrypoint for the Docker container is the binary itself. That said though, looking at the terraform docs for typical `goreleaser`-based releases and our own `.goreleaser.yaml`, it looks like the `PASSPHRASE` and most of the GPG-related env variables get leveraged prior to running through the `goreleaser` pipeline, namely:

https://github.com/hashicorp/terraform-provider-scaffolding-framework/blob/0773a77924eb125d1b508d52bd20784e8cb0df8e/.github/workflows/release.yml#L28-L33

I don't see anything like that in our pipeline here, so wondering if GPG signing actually works at this point without adding that stuff or if our GPG key just has no passphrase?

Reply:
Yeah and you can see here me using a pretty similar setup for goreleaser as a github action

https://github.com/redpanda-data/terraform-provider-redpanda/blob/ef8b975be798b35706ae2b8756ee4928522515fa/.github/workflows/release.yml

And again in the docs 

https://goreleaser.com/ci/actions/?h=gpg#signing

I was hoping to use goreleaser's stdin field in lieu of a command array but 🤦‍♀️  may as well just use make instead of both